src/components/FormComponent.vue

```diff
@@ -35,6 +63,25 @@
       return {
         state: {}
       }
+    },
+    methods: {
+      validate (field) {
+        let valid = true
+
+        if (field.validators) {
+          field.validators.forEach(validator => {
+            // validate with all the data in the form
+            valid = validator(this.data)
+          })
+        }
+        return valid
+      }
+    },
+    mixins: [VueForm],
```

Comment:
A suggest we keep these properties at the top of the file. As a side note i suggest to move these types of changes to a separate PR so we can comment on we idea itself instead of it blocking the main PR